terms/Data_Management

```diff
@@ -0,0 +1,3 @@
+# Data Management
```

Comment:
File name convention is slugified-kebab-case so the URLs look clean. I've fixed this in the main branch and cherry picked this contribution, but I've lost your authorship somehow @kilojoules when I was picking the commit so it looks like my contribution. Sorry, hope that's OK.

Reply:
OK. I'm having trouble finding some pages in my branches, these names are confusing me.